plugins/action/action.cpp

```diff
@@ -42,6 +42,11 @@ Action::Result Action::return_to_launch() const
     return _impl->return_to_launch();
 }
 
+Action::Result Action::transition_to_fw() const
```

Comment:
I'm tempted to suggest to rename this to `transition_to_fixedwing`. The goal is to have an easy to use, and self explanatory API. I would error on the side of verbosity.